### PR TITLE
feat: move to new library ci/cd workflow

### PR DESCRIPTION
**Pull request by hershmire**
jira PLAT-87

BREAKING CHANGE: switch to using yarn instead of npm. Force a new major release to officially get on the semver release track.

-----

This PR will move this library over to SemVer and fully automate the release process. For more details, look at the documentation here: https://residenetwork.atlassian.net/wiki/spaces/ENG/pages/1173291063/NPM+Library+CI+CD+Workflow

### CODE REVIEW

**Review by abuecker — APPROVED** [review]

**Review by fastestOrange — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -83,40 +81,17 @@
...
     "prettier": "^1.14.3",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
     "react-test-renderer": "^16.5.2",
-    "release-script": "^1.0.2",
     "rimraf": "^2.6.1",
-    "semantic-release": "^15.9.16",
-    "semantic-release-alt-publish-dir": "^2.1.1",
     "sinon": "^6.3.4",
     "travis-deploy-once": "^5.0.8",
```
Comment (mrudling) [inline package.json]: Looks like Travis config was removed - do we still need this dependecy.  Looks like it has been deprecated anyway - https://www.npmjs.com/package/travis-deploy-once

Comment (mrudling) [reply package.json]: I see also that the command was removed from the scripts node in package.json so it seems certain that we can remove this.

Comment (hershmire) [reply package.json]: No we shouldn't. I'll remove that as well. Nice catch!

Comment (hershmire) [reply package.json]: This has now been removed.

**Review by mrudling — APPROVED** [review]

**Review by petermiles — APPROVED** [review]

**Pull request merged**